src/main/java/com/autotune/analyzer/utils/AnalyzerErrorConstants.java

```diff
@@ -244,6 +245,8 @@ public DeleteMetricProfileAPI() {
             public static final String MISSING_METRIC_PROFILE_NAME_MSG = "Missing metric profile 'name' parameter";
             public static final String DELETE_METRIC_PROFILE_FROM_DB_FAILURE_MSG = "Failed to delete metric profile from DB: %s";
             public static final String DELETE_METRIC_PROFILE_FAILURE_MSG = "Failed to delete the specified metric profile data: %s";
+            public static final String DELETE_METRIC_PROFILE_ENTRY_NOT_FOUND_WITH_NAME = "KruizeMetricProfileEntry not found with metric profile name: ";
```

Comment:
@shreyabiradar07 - Please include tests for these

Reply:
@shreyabiradar07 - Are tests included for delete API?

Reply:
Tests are not yet included for delete API, for the 400 API response error status, there is no message being returned,  currently debugging the issue.

Reply:
Please raise a separate PR for delete API tests.